dd-sdk-android/src/main/kotlin/com/datadog/android/core/internal/data/file/FileReader.kt

```diff
@@ -71,7 +70,17 @@ internal class FileReader(
     private fun readNextFile(): Pair<File?, ByteArray> {
         val file = lockAndGetFile()
         return if (file != null) {
-            file to file.readBytes(withPrefix = prefix, withSuffix = suffix)
+            val data = try {
+                file.readBytes(withPrefix = prefix, withSuffix = suffix)
+            } catch (e: FileNotFoundException) {
+                sdkLogger.e("Couldn't create an input stream from file ${file?.path}", e)
+                ByteArray(0)
+
+                // } catch (e: IOException) {
```

Comment:
?

Reply:
Oups, some remnants of some tests I did, trying to generate such an exception in tests.